components/homme/src/share/cxx/KernelVariables.hpp

```diff
@@ -8,6 +8,7 @@
 #define KERNEL_VARIABLES_HPP
 
 #include "Types.hpp"
+#include "scream_kokkos_utils.hpp"
```

Comment:
Can the file be renamed simply `kokkos_utils.hpp`? There is no 'scream' in homme, and it helps keeping things separated. If/when scream gets into e3sm, we can think of a place where to put this utils that can be shared across different components (perhaps `components/share`).

Reply:
Yes.